admin_scripts/ubuntu/DOCUMENTATION.txt

```diff
@@ -225,14 +233,15 @@ Log ud
 *Parametre:*
     ingen
 
-Det første af disse scripts anbringer en "Log Ud"-knap på skrivebordet. Det
-eneste, den gør, er at logge ud.
+Det første af disse scripts anbringer en "Log Ud"-knap på skrivebordet.
+Det eneste, den gør, er at logge ud.
 
 Det andet script er Aarhus-specifikt; det anbringer en "Log ud"-knap på
-skrivebordet og sørger for, at brugeren altid logges af Internettet ved logout.
+skrivebordet og sørger for, at brugeren altid logges af Internettet ved
+logout.
 
-Findes under "Globale Scripts" som hhv. "Logout-knap" og "Logout-knap til
-Aarhus".
+Findes under "Globale Scripts" som hhv. "Logout-knap" og "Logout-knap
+til Aarhus".
```

Comment:
(More generally, how current _is_ this file? I don't remember any of these scripts...)